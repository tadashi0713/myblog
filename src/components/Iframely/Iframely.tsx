import React from "react";
import Helmet from "react-helmet";

export default class Iframely extends React.Component {
  componentDidMount() {
    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
    // @ts-ignore
    if (window.iframely) {
      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
      // @ts-ignore
      window.iframely.load();
    }
  }

  render() {
    return (
      <Helmet>
        <script type="text/javascript" src="https://cdn.iframe.ly/embed.js" />
      </Helmet>
    );
  }
}
