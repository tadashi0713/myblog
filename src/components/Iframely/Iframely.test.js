import React from 'react';
import renderer from 'react-test-renderer';
import Iframely from './Iframely';

describe('Iframely', () => {

  it('renders correctly', () => {
    const tree = renderer.create(<Iframely />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
