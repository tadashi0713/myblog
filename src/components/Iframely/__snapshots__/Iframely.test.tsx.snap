// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iframely renders correctly 1`] = `null`;
